recipes/dsfmt/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
+make std
+make std-check
+make sse2-check
+
+cp dSFMT.h ${PREFIX}/include
```

Comment:
No `make install`? Is there a library or something that needs to be installed as well?

Reply:
The `Makefile` doesn't have an `install` target. In the `dsfmt.mk` file in the julia repo I see that they build a library:

```
$(BUILDDIR)/dsfmt-$(DSFMT_VER)/build-compiled: $(BUILDDIR)/dsfmt-$(DSFMT_VER)/source-extracted
	cd $(dir $<) && \
	$(CC) $(CPPFLAGS) $(DSFMT_CFLAGS) $(LDFLAGS) dSFMT.c -o libdSFMT.$(SHLIB_EXT)
```

But I don't see any way to build the library directly from the `Makefile`

Reply:
That translates to roughly:

```
gcc -O3 -finline-functions -fomit-frame-pointer -DNDEBUG -DDSFMT_MEXP=19937 \
  -fPIC -fno-strict-aliasing --param max-inline-insns-single=1800 -Wmissing-prototypes \
  -Wall -std=c99 -shared dSFMT.c -o libdSFMT.so
```

Reply:
Ok. Might need to create the directory first.

Reply:
Alright, let's add it then. We may come back and tweak it later.